### ISSUE TITLE
[Q] What he point of doing test write in shared memory?

### ISSUE THREAD

**Issue by Kein** [mattolenik/NativeInjector#1]
https://github.com/mattolenik/NativeInjector/blob/c0e779927cad78da66d00ee277ccabdd43b8b33f/NativeInjector/Injector.cs#L78

Random map file created that has 0 relevance to actual injection logic and mechanism that is done through interop. What was the point of it?

**Comment by mattolenik** [comment]
That's not random, that's for writing a payload and making it available to the injected code. In my project I used this to pass a string and make it available inside the other process.

**Comment by Kein** [comment]
Yeah, I've already figured it out, thanks! Thought it was some random edge-case workaround for injection issues, like not enough COM shared memory allocated so it throws
